Homework_11/requirements.txt

```diff
@@ -0,0 +1,5 @@
+certifi==2024.2.2
+charset-normalizer==3.3.2
+idna==3.6
+requests==2.31.0
+urllib3==2.2.1
```

Comment:
requirements.txt should be in your main directory for project

Reply:
As I understand it, I should do homework in one directory, and the “requirements” folder should be in the main directory?